src/main/java/space/obminyashka/items_exchange/repository/SubcategoryRepository.java

```diff
@@ -19,4 +21,8 @@
     List<Long> findExistingIdByCategoriesId(List<Long> categoriesId, List<Long> subcategoriesId);
 
     Optional<Subcategory> findById(long id);
+
+    @Modifying
+    @Query("delete FROM Subcategory s WHERE s.id=:id")
+    void deleteById(Long id);
```

Comment:
why you use @Modifying ?

Reply:
The @Modifying annotation is used to enhance the @Query annotation so that we can execute not only SELECT queries, but also INSERT, UPDATE, DELETE. Without @Modifying command delete doesn`t work.

Reply:
great, approve